### PR TITLE
WRR-3610: Update major versions for dependencies

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [X] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
I updated the following dependencies:

* chroma-js from ^2.4.2 to ^3.1.0 (https://github.com/gka/chroma.js/releases)
No breaking change for our project.
* css-loader from ^6.11.0 to ^7.1.2 (https://github.com/webpack-contrib/css-loader/releases)
Has breaking change for our project. Needs node 18.12 or higher.
* less-loader from ^11.1.4 to ^12.2.0 (https://github.com/webpack-contrib/less-loader/releases)
Has breaking change for our project. Needs node 18.12 or higher.
* markdown-it from ^13.0.2 to ^14.1.0 (https://github.com/markdown-it/markdown-it/blob/master/CHANGELOG.md)
No breaking change for our project.
* postcss-loader from ^7.3.4 to ^8.1.1 (https://github.com/webpack-contrib/postcss-loader/releases)
Has breaking change for our project. Needs node 18.12 or higher.
* readdirp from ^3.6.0 to ^4.0.1 (https://github.com/paulmillr/readdirp/releases)
No breaking change for our project.

Minimum node version was updated from 18 to 18.12.0

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-3610

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact docs-utils module, newest changes on the top.
 
+## [unreleased]
+
+- Changed to require node 18.12 or later.
+- Updated dependencies.
```
Comment (adrian-cocoara-lgp) [inline CHANGELOG.md]: this can be deleted

Comment (alexandrumorariu) [reply CHANGELOG.md]: Done

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**